launch_testing/launch_testing/__init__.py

```diff
@@ -0,0 +1,72 @@
+import io
+import re
+from launch.output_handler import LineOutput
+
+
+class InMemoryHandler(LineOutput):
+
+    '''
+    @param name: Name of the process being tested.
+    @param ld: L{LaunchDescriptor} object that contains the processes in the test.
+    @param expected_lines: A list of lines to match the output literally or a regular expression
+        that will only need one line to match, instead of the entire output.
+    @param regex_match: If true, treat the expected_lines as a regular expression in match
+        accordingly.
+    @param filtered_prefixes: A list of prefixes that will be ignored. By default the output of
+        RTI Connext will be ignored.
+    '''
+    # TODO(esteve): This requires internal knowledge about the rmw implementations available.
+    def __init__(
+        self, name, ld, expected_lines, regex_match=False,
+        filtered_prefixes=[
```

Comment:
This cannot use a list as a default value. Instead use `None` and set the default value in the constructor.